caterva2/tests/test_api.py

```diff
@@ -733,6 +733,31 @@ def test_expr_from_expr(auth_client):
     np.testing.assert_array_equal((a[:] + 1) * 2, c[:])
 
 
+def test_expr_no_operand(auth_client):
```

Comment:
Is that the test for this PR?  I was expecting something to test the new `compute=` arg in `lazyexpr()` func.  Perhaps this PR is not ready yet?